plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultQueryBuilder.java

```diff
@@ -340,6 +342,13 @@ protected String formatProjections(JdbcClient client, Map<JdbcColumnHandle, Stri
                 .collect(joining(", "));
     }
 
+    protected String formatProjectionAliases(JdbcClient client, String relationAlias, Map<JdbcColumnHandle, String> projections)
+    {
+        return projections.values().stream()
+                .map(s -> format("%s.%s", relationAlias, client.quoted(s)))
```

Comment:
no need to qualify then with relationAlias
this used in JOIN context where left and right relations had all aliases disambiguated with projections

Reply:
Done!